middleware.go

```diff
@@ -10,11 +10,24 @@ import (
 	"github.com/pressly/chi/middleware"
 )
 
-// RequestLogger is a middleware for the github.com/Sirupsen/logrus to log requests.
-// It is equipt to handle recovery in case of panics and record the stack trace
-// with a panic log-level.
+type RequestLoggerConfig struct {
+	// Logger is the backing logger to log to.
+	Logger *logrus.Logger
+	// WriteRequestStartedLine indicates if a request started line should be written or not.
+	// If false, only a request completed line will be written.
+	WriteRequestStartedLine bool
```

Comment:
Let's just remove the `"request started"` line for good. We don't really need it. We don't need the toggle either.

If anyone needs to log this line for some reason, they can re-add it as their own middleware.